learning_assistant/urls.py

```diff
@@ -4,14 +4,19 @@
 from django.urls import re_path
 
 from learning_assistant.constants import COURSE_ID_PATTERN
-from learning_assistant.views import CourseChatView
+from learning_assistant.views import CourseChatView, LearningAssistantEnabledView
 
 app_name = 'learning_assistant'
 
 urlpatterns = [
     re_path(
-        fr'learning_assistant/v1/course_id/{COURSE_ID_PATTERN}',
+        fr'learning_assistant/v1/course_id/{COURSE_ID_PATTERN}$',
```

Comment:
This `$` tells Django to match the regular expression fully; see [documentation](https://docs.djangoproject.com/en/5.0/ref/urls/#re-path). Without this symbol, a call to `learning_assistant/v1/course_enabled/<course_id>/enabled` would be matched by this regular expression and map to the wrong view.